roles/baseline/tasks/main.yml

```diff
@@ -7,10 +7,9 @@
     efsFileSystemId: "{{ V4_CFG_EFS_FSID }}"
     awsRegion: "{{ V4_CFG_EFS_REGION }}"
   when:
-    - PROVIDER is defined
-    - PROVIDER == "aws"
     - V4_CFG_MANAGE_STORAGE is defined
     - V4_CFG_MANAGE_STORAGE
+    - CREATE_EFS_STORAGE_CLASS is defined
```

Comment:
Why the new variables CREATE_EFS_STORAGE_CLASS and CREATE_NFS_STORAGE_CLASS? They are unneeded and don't following the naming standard.